errors.txt

```diff
@@ -0,0 +1,202 @@
+15) CodeIgniter\Files\FileCollectionTest::testResolveDirectorySymlink
+ErrorException: symlink(): Permission denied
```

Comment:
To bypass the issue related to Permission denied rights, you can run your terminal with "Run as Administrator" privileges.

Reply:
I know about it.

Reply:
You can skip the test when in windows by adding a `@requires osfamily linux` phpdoc

Reply:
Yes, but how for Github actions?